src/Interactive/Host/Interactive/Core/InteractiveHost.RemoteService.cs

```diff
@@ -28,14 +31,16 @@ internal sealed class RemoteService
             private Thread _readErrorOutputThread;      // nulled on dispose
             private volatile ProcessExitHandlerStatus _processExitHandlerStatus;  // set to Handled on dispose
 
-            internal RemoteService(InteractiveHost host, Process process, int processId, Service service)
+            //(miziga) should there be a jsonRpc object in parameters?
+            internal RemoteService(InteractiveHost host, Process process, int processId, JsonRpc jsonRpc)
             {
                 Debug.Assert(host != null);
                 Debug.Assert(process != null);
-                Debug.Assert(service != null);
+                //(miziga) can you even check if a jsonRpc is null? necessary?
+                //Debug.Assert(jsonRpc != null);
```

Comment:
Let's keep the assert here for now. We can later annotate this file with nullable references, but let's not do that in this PR.